htdocs/expedition/class/expedition.class.php

```diff
@@ -732,17 +732,17 @@ public function valid($user, $notrigger = 0)
 
 		// Define new ref
 		if (!$error && (preg_match('/^[\(]?PROV/i', $this->ref) || empty($this->ref))) { // empty should not happened, but when it occurs, the test save life
```

Comment:
The condition $error is not necessary here (its value is 0)
And it makes a difficulty to understand "else" condition.
No correction is needed here but it's just a remark.